server/main.go

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2023 OpenIMSDK open source community. All rights reserved.
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/OpenIMSDK/OpenKF/server/internal/config"
+	"github.com/OpenIMSDK/OpenKF/server/internal/db"
+	"github.com/OpenIMSDK/OpenKF/server/internal/router"
+	"github.com/OpenIMSDK/OpenKF/server/pkg/log"
+	"github.com/OpenIMSDK/OpenKF/server/pkg/server"
+)
+
+func init() {
+	// init config
+	config.ConfigInit()
+
+	// logo
+	_logo := `
```

Comment:
<img src="https://media0.giphy.com/media/3oEduOnl5IHM5NRodO/giphy.gif"/>
Can you can, go to https://github.com/avelino/awesome-go to find more pattern, command line beautification output what ha ha ha ha ha

Reply:
Sounds good, I will try to choose one.